.github/workflows/default_branch_verification.yml

```diff
@@ -6,19 +6,22 @@ on:
 
 jobs:
   build:
+    runs-on: ubuntu-latest
+    container: gradle:8.0.0-jdk17
     steps:
       - name: Build
         uses: ./.github/workflows/setup_build_environment.yml
         with:
           command: ./gradlew build
-        secrets: inherit
```

Comment:
It's not needed anymore?

Reply:
I'm not sure, but it's not allowed when using `steps` that's why I've deleted it. After merging it the develop build should pass because the syntax will be correct but I'm not sure if it will work correctly. I will try to test it on a separate branch later.

Reply:
You can just switch temporarily this workflow to trigger on pull request and check if everything will pass

Reply:
It's not passing 😅 changing to draft